### ISSUE TITLE
Font image and text effect replacement in chat causes lag at high frequencies

### ISSUE THREAD

**Issue by Projecki** [PluginBugs/Issues-ItemsAdder#3792]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### What happened?

ItemsAdder is causing lag:
https://spark.lucko.me/MS1XdG5W3u
view plugins 

### Steps to reproduce the issue

n/a

### Server version

This server is running UniverseSpigot version git-UniverseSpigot-"1954a51" (MC: 1.20.4) (Implementing API version 1.20.4-R0.1-SNAPSHOT) (Git: 1954a51 on ver/1.20.4-stable)
Universe-git-UniverseSpigot-"1954a51" (MC: 1.20.4)

### ItemsAdder Version

[18:17:40 INFO]: ItemsAdder version 3.6.4-release-2
[18:17:40 INFO]: Customized Minecraft without client mods!
[18:17:40 INFO]: Website: devs.beer
[18:17:40 INFO]: Author: LoneDev

### ProtocolLib Version

[18:18:06 INFO]: ProtocolLib version 5.2.1-SNAPSHOT-689
[18:18:06 INFO]: Provides read/write access to the Minecraft protocol.
[18:18:06 INFO]: Authors: dmulloy2 and comphenix

### LoneLibs Version

[18:17:53 INFO]: LoneLibs version 1.0.54
[18:17:53 INFO]: Collection of libs used by my plugins
[18:17:53 INFO]: Authors: LoneDev and others

### Error (optional)

```shell
n/a
```

**Comment by Andre601** [comment]
You're using an outdated version of ItemsAdder for 1.20.4, please update it.

**Comment by Andre601** [comment]
In addition did you not provide the server logs, as requested. And as a final note are you using some random fork that IA is not tested against, so it could also be an issue with them messing things up.

**Comment by Andre601** [comment]
And as some final notes:

- You assign your server 28GB! No Server needs this much RAM to function and such high values can have a negative influence on the JVM too.
- Looking at the Plugin info, IA barely uses any of the tick (a bit more than 11%) and going down the tree shows that it goes to ProtocolLib at the end, so this is most likely a fault on their end.
- Going down the Flat list, the most heavy used stuff are server tick actions, which are nothing unusual. Also, going down the first entry shows MythicMobs at the end being the first plugin to show.

tl;dr: Barely anything in your Spark report hints at Spigot really being at fault here.

**Comment by LoneDev6** [comment]
Based on that:
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/27242001/3ed7e6fa-22f3-4a62-9f7b-3e920ae519c3)

Seems to be related to the text effects and font images replacement in chat.
You probably didn't set a limiter to the chat so people can spam the chat very fast.
What you can do is:
- limit the chat messages by installing a chat manager to add some limit to the amount of chat messages a player can send
- disable font images and text effects replacement in config.yml of ItemsAdder

I find it quite strange to be honest because I already optimized that code as much as possible, so seems caused by the amount of messages sent rather than my implementation performance.

**Comment by LoneDev6** [comment]
Duplicate of: #3792
